File: final_project/GroupFinalAPI/Models/usedCars.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const UsedCar = new Schema({

    year:
    {
        type: Number,
        required: true
    },
    make:
    {
        type: String,
        required: true
    },
    model:
    {
        type: String,
        required: true
    },
    miles:
    {
        type: String,
        required: true
    },
    price:
    {
        type: String,
        required: true
    },
    
});
mongoose.model('UsedCar', UsedCar);

File: final_project/lib/Models/usedCars.dart

class UsedCars {
  final String id;
  final int year;
  final String make;
  final String model;
  final String miles;
  final String price;


  UsedCars._(this.id, this.year, this.make, this.model, this.miles, this.price);
  
  factory UsedCars.fromJson(Map json) {
    final id = json[''].replaceAll('ObjectId(\"', '').replaceAll('\")', '');
    final year = json['year'];
    final make = json['make'];
    final model = json['model'];
    final miles = json['miles'];
    final price = json['price'];
    return UsedCars._(id, year, make, model, miles, price);
  }
}